ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/TestKsqlRestApp.java

```diff
@@ -401,6 +402,9 @@ private static KsqlRestConfig buildConfig(
     configMap.put(KsqlConfig.KSQL_STREAMS_PREFIX + "cache.max.bytes.buffering", 0);
     configMap.put(KsqlConfig.KSQL_STREAMS_PREFIX + "auto.offset.reset", "earliest");
     configMap.put(KsqlConfig.KSQL_ENABLE_UDFS, false);
+    configMap.put(KsqlRestConfig.KSQL_HEARTBEAT_SEND_INTERVAL_MS_CONFIG, 600000);
```

Comment:
this is so that it does not heartbeat by default?

Reply:
Yes, else the tests fail as they have a disables ksql client that is not allowed to do requests